build/Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM docker.io/openshift/origin-release:golang-1.16 as build
 LABEL stage=build
 
+# dos2unix is needed to build CNI plugins
```

Comment:
Should/could this commit + the CNI plugins update go into a separate PR?
Theres a bit more being done here than normal for a submodule update.

Reply:
Sure. I can break this off into a separate PR.

Reply:
Done. https://github.com/openshift/windows-machine-config-operator/pull/841